pi5/board.go

```diff
@@ -224,7 +224,8 @@ func (b *pinctrlpi5) DigitalInterruptByName(name string) (board.DigitalInterrupt
 		Name: name,
 		Pin:  name,
 	}
-	interrupt, err := b.boardPinCtrl.NewDigitalInterrupt(defaultInterruptConfig, mapping, nil)
+
+	interrupt, err := b.boardPinCtrl.NewDigitalInterrupt(defaultInterruptConfig, mapping, 0, nil)
```

Comment:
we should consider making settingDebounce an API in the future so someone can do it in a script

Reply:
true ^

is there any way to add a debounce timer as is, since we're automatically setting it to 0 and disabling it here?

Reply:
so the pi5 implementation in this repo is only for testing pinctrl logic. in the raspberry-pi repo users have access to debounce through the pin configs